AnimatedWordCloud/Animator/TimelapseWordVector.py

```diff
@@ -107,7 +171,7 @@ def __init__(self) -> None:
         """
 
         # main data
-        self.timeframes: List[TimeFrame] = []
+        self.timeframes: list[TimeFrame] = []
 
     def __getitem__(self, index: int) -> TimeFrame:
```

Comment:
timeframes's type is list, so this method works if index's type is slice.
Fix it if it is concerning to you.

Reply:
I don't think it needs to be fixed.

Reply:
Good point. I added `list` annotation